std_msgs/Header header
int32 current_pos
int32 target_pos
int32 current_pos_pot
int32 current_pos_enc
bool fault_status
bool enable